### DISCUSSION TITLE
Internet Explorer glossary page

### DISCUSSION THREAD

**Discussion by MrBrain295** [mdn/mdn-community#155] [Content]
There is a glossary page for internet explorer but, internet explorer died. Should it be removed?

**Comment by teoli2003** [comment]
A small update may be needed, but I don't think this page adds a significant burden to keep updated, nor makes it harder for web devs to write web site or understand how to do it.

So I would say, let's keep it and eventually update it a bit to mention the end of the support on desktop computers.

**Reply by Rumyra** [reply]
Agree with @teoli2003 👍
